docs/tutorials/mlflow.ipynb

```diff
@@ -0,0 +1,774 @@
+{
```

Comment:
<p>Since you discussed the tracking options previously, this would be a good place to explain what happens - this notebook is taking the "easy way out" just looking at the run object for outputs/results, but it might be a good idea to put a screenshot of the UI or something, or maybe even show the name of train_run and explain how it correlates to the MLFlow experiment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/4694/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>i agree</p>

Reply:
@ZeevRispler can you add details as Saar requested?